app/lib/folio/mailer_base.rb

```diff
@@ -40,4 +40,18 @@ def compiled_asset_contents(key)
       File.read(File.join(Rails.application.assets_manifest.directory, manifest_file))
     end
   end
+
+  def with_user_locale(user, locale: nil)
+    locale ||= user.preferred_locale || user.auth_site.locale
```

Comment:
tady bych jeste dal `user.auth_site.try(:locale) || I18n.locale`, protoze to jinak nekde zarucene padne